tinygrad/renderer/assembly.py

```diff
@@ -58,12 +59,9 @@ def load_store_ptr_arithmetic(x:UOp, buf:UOp, alu:Optional[UOp]=None, const:Opti
 ])
 
 class PTXRenderer(Renderer):
-  device = "CUDA"
```

Comment:
Why was this removed?

Reply:
I removed it as it was already as default on the constructor and nowhere in the codebase was accesses statically (`PTXRenderer.device`).